hack/terraform-quickstart/resources.tf

```diff
@@ -0,0 +1,49 @@
+provider "aws" {
+  access_key = "${var.access_key_id}"
+  secret_key = "${var.access_key}"
+  region     = "us-east-1"
+}
+
+resource "aws_instance" "bootstrap_node" {
+  ami           = "${data.aws_ami.coreos_ami.image_id}"
+  instance_type = "m3.medium"
+  key_name      = "${var.ssh_key}"
+
+  tags {
+    Name = "${var.instance_tags}"
+  }
+}
+
+resource "aws_instance" "worker_node" {
```

Comment:
Ah this will make it simpler to set a variable that determines the number of worker nodes.